bundles/org.openhab.binding.mail/src/main/feature/feature.xml

```diff
@@ -4,6 +4,7 @@
 
     <feature name="openhab-binding-mail" description="Mail Binding" version="${project.version}">
         <feature>openhab-runtime-base</feature>
+        <feature>openhab-core-automation</feature>
```

Comment:
Imho this isn't the right fix.
The new rule engine must not be a prerequisite for using this binding.
Afaik, it currently only has an optional import to the annotations and nothing else.

I haven't dived into your analysis in the issue though, I have to admit.

Reply:
I don't understand it fully, too. 

But without this feature (or one of it's bundles) sending multi-part mails is not possible. Installing this feature made it work in two different setups.

Reply:
Maybe it is some dependency that the new rule engine brings with it, which is missing. A specific version of Apache Commons or whatever. I think we have to investigate it a bit further...